pkg/file/file.go

```diff
@@ -0,0 +1,43 @@
+package file
+
+import (
+	"io/ioutil"
+
+	isopod "github.com/cruise-automation/isopod/pkg"
+	"go.starlark.net/starlark"
+)
+
+type filePackage struct {
+	*isopod.Module
+}
+
+// New returns a new starlark.HasAttrs object for file package.
+func New() starlark.HasAttrs {
+	f := &filePackage{}
+
+	f.Module = &isopod.Module{
+		Name: "file",
+		Attrs: starlark.StringDict{
+			"read": starlark.NewBuiltin("file.read", f.readFile),
+		},
+	}
+
+	return f
+}
+
+// readFile blindly reads a file and returns it as a string
+func (f *filePackage) readFile(t *starlark.Thread, b *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var filename string
+	unpacked := []interface{}{
+		"filename", &filename,
+	}
+
+	if err := starlark.UnpackArgs(b.Name(), args, kwargs, unpacked...); err != nil {
+		return nil, err
+	}
+	dat, err := ioutil.ReadFile(filename)
```

Comment:
I guess this gets the work done, but `filename` here is either absolute (which is not user-friendly since different people clone the same configuration repo at different path) or relative to the current working dir (which yields surprising results, since `load()` function in starlark is relative to the current starlark file enclosing this particular `load()`.)

Perhaps you could hack together something that ensure the relativity is interpreted consistently in the same file by changing https://github.com/cruise-automation/isopod/blob/master/pkg/loader/loader.go.

Also, would be great to add some tests so we are use this implementation is correct.

Reply:
@cxuu do you mean to merge the behavior into the loader or to handle relative paths the same way that is done there?

Reply:
I was trying to say that `load()` and `file.read()` should "handle relative paths the same way". You could reuse if possible the code in loader.go